src/components/Dashboard/StaffNotation.vue

```diff
@@ -335,7 +327,7 @@ export default class StaffNotation extends Vue {
 
   @Emit('close')
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  private close (needReload: boolean): void {}
+  close (needReload: boolean): void { /* no empty function */ }
```

Comment:
There are a lot of these in the code. This is done to fix the `Unexpected empty method` SonarCloud code smell.

For reference: https://eslint.org/docs/latest/rules/no-empty-function